src/otx/core/model/entity/base.py

```diff
@@ -213,7 +214,7 @@ def export(
         Returns:
             Path: path to the exported model.
         """
-        return self._exporter.export(self.model, output_dir, base_name, export_format, precision)
+        return self._exporter.export(self.model, output_dir, base_name, export_format, precision, dump_auxiliaries)
```

Comment:
When dump_auxiliaries is true, we need a check here, if custom forward actually exists or not

Reply:
Done